pandas/core/format.py

```diff
@@ -241,7 +242,10 @@ def to_string(self):
             fmt_values.insert(row_num + n_header_rows, dot_str)
             fmt_index.insert(row_num + 1, '')
 
-        result = self.adj.adjoin(3, *[fmt_index[1:], fmt_values])
+        if self.index:
+            result = self.adj.adjoin(3, *[fmt_index[1:], fmt_values])
+        else:
```

Comment:
Dataframe does not use adjoin. In dataframe we use a [list](https://github.com/pydata/pandas/blob/784445cbe81cc8b2da4e2324689f175cb2a2edbf/pandas/core/format.py#L492) and insert index into that. I was expecting that dataframe and series formatter code will be similar but this is not the case .